docs/requirements.txt

```diff
@@ -1,5 +1,6 @@
 sphinx > 3.0.0
-sphinx-rtd-theme == 0.5.0
-sphinx-argparse == 0.2.5
-nbsphinx == 0.8.5
-sphinx_copybutton == 0.3.1
+sphinx-rtd-theme
+sphinx-argparse 
+nbsphinx >0.8.5
+sphinx_copybutton
+ipython_genutils
```

Comment:
see the notes here: https://github.com/ipython/ipython_genutils
Is this necessary? Or do we just need to adjust the required versions for the other stuff?

Reply:
I am not sure what to change version wise besides jinja to revert back to 2.1.2 or whatever we had before. This works for now but I will see if I can get it to work without explicitly relying on this package